packages/ledger-amino/src/launchpadledger.ts

```diff
@@ -44,13 +44,10 @@ export class LaunchpadLedger {
       prefix: cosmosBech32Prefix,
       testModeAllowed: false,
     };
-    const { hdPaths, prefix, testModeAllowed } = {
-      ...defaultOptions,
-      ...options,
-    };
-    this.testModeAllowed = testModeAllowed;
-    this.hdPaths = hdPaths;
-    this.prefix = prefix;
+
+    this.testModeAllowed = options.testModeAllowed ?? defaultOptions.testModeAllowed;
+    this.hdPaths = options.hdPaths ?? defaultOptions.hdPaths;
+    this.prefix = options.prefix ?? cosmosBech32Prefix;
```

Comment:
With this change, `defaultOptions.prefix` is unused. Let's either put all defaults here or collect all defaults in `defaultOptions`. Both ways are fine for me but I have a slight tendency towards just removing `defaultOptions`

Reply:
just my mistake: 8042c252e

Reply:
just my mistake: 8042c25